src/App.tsx

```diff
@@ -2,15 +2,23 @@ import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
+import React, { useState } from 'react';
+import { Movie } from './types/Movie';
 
 export const App = () => {
+  const [posts, setPosts] = useState<Movie[]>([...moviesFromServer]);
+
+  const onAdds = (movie: Movie) => {
```

Comment:
```suggestion
  const handleAddMovie = (movie: Movie) => {
```
- on add what?
- handler names should start from `handle` https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364